service/framework.go

```diff
@@ -383,12 +414,47 @@ func NewVersionedResources(config Config, k8sClient kubernetes.Interface, awsCon
 
 	// We create the list of resources and wrap each resource around some common
 	// resources like metrics and retry resources.
-	var resources []framework.Resource
+	var resourcesV1 []framework.Resource
+	{
+		resourcesV1 = []framework.Resource{
+			kmsKeyResource,
+			s3BucketResource,
+			s3BucketObjectResourceV1,
+			cloudformationResource,
+			namespaceResource,
+			serviceResource,
+			endpointsResource,
+		}
+
+		// Disable retry wrapper due to problems with the legacy resource.
+		//
+		// NOTE that the retry resources wrap the underlying resources first. The
+		// wrapped resources are then wrapped around the metrics resource. That way
+		// the metrics also consider execution times and execution attempts including
+		// retries.
+		/*
+			retryWrapConfig := retryresource.DefaultWrapConfig()
+			retryWrapConfig.Logger = config.Logger
+			cloudFormationResources, err = retryresource.Wrap(cloudFormationResources, retryWrapConfig)
+			if err != nil {
+				return nil, microerror.Mask(err)
+			}
+		*/
```

Comment:
There is actually no legacy resource here. I think this can be removed then. Same for below.

Reply:
Done.